packages/svelte/src/compiler/phases/2-analyze/visitors/RegularElement.js

```diff
@@ -77,9 +77,7 @@ export function RegularElement(node, context) {
 
 	if (
 		node.attributes.some(
-			(attribute) =>
-				attribute.type === 'Attribute' &&
-				(attribute.name === 'autofocus' || attribute.name === 'muted')
+			(attribute) => attribute.type === 'Attribute' && cannot_be_set_statically(attribute.name)
```

Comment:
This logic can move to `Attribute` visitor rather than be here.

Reply:
I just strictly did replace the usage sites, nothing else. This is meant to be a consolidation PR, nothing else